imagepipeline-base/src/main/java/com/facebook/imagepipeline/transcoder/DownsampleUtil.java

```diff
@@ -15,6 +15,7 @@
 import com.facebook.imagepipeline.common.RotationOptions;
 import com.facebook.imagepipeline.image.EncodedImage;
 import com.facebook.infer.annotation.Nullsafe;
+import java.text.DecimalFormat;
```

Comment:
```suggestion
import java.text.DecimalFormat;
```suggestion
import java.text.DecimalFormat;

```

```